src/org/labkey/test/tests/InvalidateSessionTest.java

```diff
@@ -139,8 +125,13 @@ public void testCookieAndSessionFromLogout() throws IOException
         }
 
         log("Capture the cookie after logout");
-        Cookie afterCookie = getCookies(getCurrentUser()).get(Connection.JSESSIONID);
-        Assert.assertFalse("Before and after log out cookie should be different", beforeCookie.equals(afterCookie));
+        Set<Cookie> afterCookie = getDriver().manage().getCookies();
+        Assert.assertFalse("Before and after log out cookie should be different", getJSessionIdValue(beforeCookie).equals(getJSessionIdValue(afterCookie)));
+    }
+
+    private String getJSessionIdValue(Set<Cookie> cookies)
+    {
+        return cookies.stream().toList().get(1).getValue();
```

Comment:
Don't assume that the session cookie is at a particular index. Get the cookie named `Connection.JSESSIONID`.